apps/src/StudioApp.js

```diff
@@ -1,5 +1,14 @@
-// Globals:
-//   Blockly
+/* jshint
+  strict: true,
+  curly: true,
+  funcscope: true,
+  newcap: true,
+  nonew: true,
+  shadow: false,
+  unused: true
+*/
+/* global Blockly: true */
+'use strict';
```

Comment:
Enables ECMAScript 5 [Strict Mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) for the whole file.